guides/server/live-layouts.md

```diff
@@ -71,16 +71,16 @@ mount:
 Then access `@page_title` in the root layout:
 
 ```heex
-<title><%= @page_title %></title>
+<title>{@page_title}</title>
 ```
 
 You can also use the `Phoenix.Component.live_title/1` component to support
 adding automatic prefix and suffix to the page title when rendered and
 on subsequent updates:
 
 ```heex
-<Phoenix.Component.live_title prefix="MyApp – ">
-  <%= assigns[:page_title] || "Welcome" %>
+<Phoenix.Component.live_title default="Welcome" prefix="MyApp – ">
+  {assigns[:page_title]}
```

Comment:
My understanding is it's fine either way but maybe should use the more common syntax:

```suggestion
  {@page_title}
```

Reply:
I believe we use the other syntax because it may not always be set.

Reply:
forgot to mention, I think we can use the other other one because we set `default` attr. Otherwise agreed on `assigns[:page_title] || default`

Reply:
We could probably always use `@page_title` in the root layouts if we did

```
embed "layouts/*"

attr :page_title, :string, default: nil
def root(assigns)
```

to actually set a default, but we don't. And even if, the docs should probably not have too many assumptions, otherwise people might be confused when it doesn't work. So keeping the optional syntax gets my vote.